### ISSUE TITLE
Enforce Numpy doc style

### ISSUE THREAD

**Issue by felixhekhorn** [N3PDF/pineko#23]
Following @AleCandido s [comment in `eko`](https://github.com/N3PDF/eko/pull/118#discussion_r886701620) on now we're trying to use Numpy doc style guide - actually, we could really enforce this by [pre-commit](http://www.pydocstyle.org/en/stable/usage.html#usage-with-the-pre-commit-git-hooks-framework) using [pydocstyle](http://www.pydocstyle.org/en/stable/index.html) - shall we do this? @andreab1997 @AleCandido @giacomomagni @niclaurenti 

it would (most likely) be a major rework in `eko` (I'm guessing), so we could instead start it here: 21 issues

**Comment by AleCandido** [comment]
Consider that `pre-commit` is working incrementally: it only applies to the files you changed in your commit.

**Comment by AleCandido** [comment]
Yes, I would not try to make an enormous effort to standardize all at once: while touching the files with various changes, let's improve docstrings incrementall.

The moment we believe is almost fine, we will attempt to run `pre-commit run --all-files`

**Comment by felixhekhorn** [comment]
Introduced in https://github.com/N3PDF/pineko/pull/19/commits/918f547d80c4bf81f86f7d66fcd4241dd1a8ddf8 ; completed in https://github.com/N3PDF/pineko/pull/19/commits/20e6c5765afc43eafd05468101d54dc997631755

**Comment by AleCandido** [comment]
So much the better.

If it's done, feel free to close the issue.
